package.json

```diff
@@ -216,87 +220,83 @@
 		"typescript": "4.4.2",
 		"uglify-js": "3.13.7",
 		"unzipper": "0.10.11",
+		"utility-types": "^3.10.0",
 		"uuid": "8.3.0",
 		"wd": "1.12.1",
 		"webpack": "5.65.0",
 		"webpack-bundle-analyzer": "4.4.2",
 		"worker-farm": "1.7.0"
 	},
 	"scripts": {
-		"analyze-bundles": "npm run build -- --webpack-bundle-analyzer",
+		"analyze-bundles": "pnpm build -- --webpack-bundle-analyzer",
 		"api-docs:blocks": "node ./bin/api-docs/gen-block-lib-list.js",
 		"api-docs:ref": "node ./bin/api-docs/update-api-docs.js",
 		"api-docs:theme-ref": "node ./bin/api-docs/gen-theme-reference.js",
 		"clean:packages": "rimraf \"./packages/*/@(build|build-module|build-style)\"",
 		"clean:package-types": "tsc --build --clean",
-		"prebuild:packages": "npm run clean:packages && lerna run build",
-		"build:packages": "npm run build:package-types && node ./bin/packages/build.js",
+		"prebuild:packages": "pnpm clean:packages && pnpm run -r build",
+		"build:packages": "pnpm build:package-types && node ./bin/packages/build.js",
 		"build:package-types": "node ./bin/packages/validate-typescript-version.js && tsc --build",
 		"build:plugin-zip": "./bin/build-plugin-zip.sh",
-		"build": "npm run build:packages && wp-scripts build",
+		"build": "pnpm build:packages && wp-scripts build",
 		"changelog": "./bin/plugin/cli.js changelog",
 		"check-licenses": "concurrently \"wp-scripts check-licenses --prod --gpl2 --ignore=@react-native-community/cli,@react-native-community/cli-platform-ios\" \"wp-scripts check-licenses --dev\"",
-		"precheck-local-changes": "npm run docs:build",
-		"check-local-changes": "( git diff -U0 | xargs -0 node bin/process-git-diff ) || ( echo \"There are local uncommitted changes after one or both of 'npm install' or 'npm run docs:build'!\" && git diff --exit-code && exit 1 );",
-		"dev": "npm run build:packages && concurrently \"wp-scripts start\" \"npm run dev:packages\"",
+		"precheck-local-changes": "pnpm docs:build",
+		"check-local-changes": "( git diff -U0 | xargs -0 node bin/process-git-diff ) || ( echo \"There are local uncommitted changes after one or both of 'npm install' or 'pnpm docs:build'!\" && git diff --exit-code && exit 1 );",
```

Comment:
```suggestion
		"check-local-changes": "( git diff -U0 | xargs -0 node bin/process-git-diff ) || ( echo \"There are local uncommitted changes after one or both of 'pnpm install' or 'pnpm docs:build'!\" && git diff --exit-code && exit 1 );",
```

Reply:
Done in https://github.com/WordPress/gutenberg/pull/38624/commits/618ae4ff75845f0d106830dfe0eaa97237ecccbb